packages/shared/src/components/Feed.tsx

```diff
@@ -302,7 +293,7 @@ export default function Feed<T>({
     row: number,
     column: number,
   ) => {
-    copyLink(post, index, row, column);
+    copyLink({ post, columns: index, row, column });
```

Comment:
One of the benefits of using an object is this: the row was on the 3rd parameter, but it is supposed to be the column.

Reply:
Hehe I know that's one of the reasons I prefer Object notation (think I had a discussion with @kopancek about this a while back?)

Reply:
Yes, we discussed this on a different PR, where IIRC we only had 2 params on a function. Completely agree that object notation is the way to go here, since it's more than 2 parameters :)